main/file.c

```diff
@@ -821,7 +822,7 @@ struct ast_filestream *ast_openstream_full(struct ast_channel *chan, const char
 	if (!fileexists_core(filename, NULL, preflang, buf, buflen, file_fmt_cap) ||
 		!ast_format_cap_has_type(file_fmt_cap, AST_MEDIA_TYPE_AUDIO)) {
 
-		ast_log(LOG_WARNING, "File %s does not exist in any format\n", filename);
+		ast_debug(1, "File %s does not exist in any format\n", filename);
```

Comment:
I might be missing something, but why shouldn't this be a warning now? I think there should still be a warning if the file doesn't exist in either location.

Reply:
Yeah, you're right.  This patch came from someone else and I forgot to change that back to a warning.